src/OpenTK.Mathematics/MathHelper.cs

```diff
@@ -945,41 +934,58 @@ public static double Clamp(double n, double min, double max)
             return Math.Max(Math.Min(n, max), min);
         }
 
+        /// <summary>
+        /// Scales the specified number linearly between a minimum and a maximum.
+        /// </summary>
+        /// <remarks>If the value range is zero, this function will throw a divide by zero exception.</remarks>
+        /// <param name="value">The number to scale.</param>
+        /// <param name="valueMin">The minimum expected number (inclusive).</param>
+        /// <param name="valueMax">The maximum expected number (inclusive).</param>
+        /// <param name="resultMin">The minimum output number (inclusive).</param>
+        /// <param name="resultMax">The maximum output number (inclusive).</param>
+        /// <returns>The number, scaled linearly between min and max.</returns>
         [Pure]
-        private static unsafe int FloatToInt32Bits(float f)
+        public static int Map(int value, int valueMin, int valueMax, int resultMin, int resultMax)
```

Comment:
TransposeRange would be better as a name I think.

Map would imply somehow taking a function of `int -> int` and applying it to the value. Which is a bit peculiar.

Reply:
I think a simpler name would be `Remap`, as in "remapping a value into another range".

Reply:
The `Map` you are thinking of is in a functional programming context, to people who don't do functional `map` often means this function.